docs/configuring.md

```diff
@@ -703,6 +703,45 @@ questions with default answers.
     copier -fd 'user_name=Manuel Calavera' copy template destination
     ```
 
+!!! info
+
+    The `-d, --data` flags are mutually exclusive with the [`--data-file`](#data_file) flag.
+
+### `data_file`
+
+-   Format: `str`
+-   CLI flags: `--data-file`
+-   Default value: N/A
+
+As an alternative to [`--data`][data] you can also pass the path to a YAML file that contains your data.
```

Comment:
A newline is missing. Also, how a out adding the short flag here as well? And shouldn't the link be an anchor link?

```suggestion
As an alternative to [`-d, --data`](#data) you can also pass the path to a YAML file that contains your data.

```